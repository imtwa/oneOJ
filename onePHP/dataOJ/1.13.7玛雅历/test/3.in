186
2. yax 120
7. xul 632
14. mol 3560
18. yax 3520
13. ceh 1512
3. muan 1700
14. kankin 1148
16. pop 3548
8. yoxkin 1416
5. xul 2432
9. yoxkin 2508
1. zotz 704
3. mol 2628
18. kankin 2396
7. pop 372
6. yoxkin 964
15. zotz 700
16. zip 1128
5. tzec 3788
6. pop 628
18. mac 1104
16. muan 924
12. kankin 3128
16. kankin 3612
4. mol 2308
5. zac 284
2. cumhu 2896
2. pax 1044
12. zac 964
17. koyab 1860
14. zotz 1460
13. tzec 1144
1. kankin 3692
12. mac 1152
18. chen 3704
9. chen 248
6. pop 2128
13. kankin 2072
8. kankin 1408
4. zac 504
11. no 3428
16. yax 720
1. cumhu 2892
11. mac 324
7. yoxkin 3736
14. koyab 2416
3. yoxkin 68
18. cumhu 856
10. mol 3788
10. zip 3144
16. mac 920
13. zip 76
17. xul 516
9. mac 1744
2. zotz 2924
11. ceh 2728
14. yoxkin 2264
12. xul 2984
11. yax 2676
13. zip 708
6. pop 2040
8. chen 3124
11. no 2320
1. yax 3128
15. pop 112
17. yax 1032
16. pax 1076
9. zotz 344
7. mol 2944
13. mac 1764
15. yoxkin 44
2. mac 1632
11. tzec 1116
2. no 1144
10. xul 1360
15. mol 1960
6. ceh 2360
13. kankin 2452
8. kankin 2944
6. koyab 1104
12. zip 3128
5. kankin 1288
6. xul 3020
14. xul 3008
18. zac 992
6. mac 2300
3. zac 2740
14. zac 2028
15. muan 3908
6. zip 1400
10. kankin 3916
15. zac 3344
9. yoxkin 2448
7. zac 2400
17. mac 1400
1. pax 108
12. pax 808
3. mac 3544
8. ceh 204
4. yoxkin 1316
4. cumhu 228
5. zip 632
17. tzec 3588
17. pax 408
3. xul 2972
15. yoxkin 584
4. zac 412
12. yax 1532
14. mac 2524
18. zotz 840
4. koyab 1844
10. kankin 3748
5. ceh 3364
17. no 2068
4. pop 1684
9. tzec 268
1. chen 2816
11. kankin 2796
4. pop 1888
6. kankin 2912
10. no 2668
7. mol 1980
5. koyab 328
1. xul 2444
14. no 1688
2. zip 788
12. yax 2592
12. zac 3272
11. pop 2732
2. yoxkin 3988
17. mac 360
16. koyab 1860
5. no 1576
1. no 1328
10. zotz 2544
7. muan 3428
6. chen 2344
18. pax 960
17. xul 2208
18. yoxkin 360
9. no 2440
3. pax 1052
18. muan 2804
2. muan 3580
1. cumhu 1516
15. zip 2764
2. muan 2408
15. mol 1364
17. yoxkin 4000
6. ceh 956
11. zac 2588
2. cumhu 2768
8. ceh 2192
11. cumhu 1788
5. kankin 3284
4. no 2184
9. zip 1716
17. zotz 1288
13. zip 220
5. zip 796
6. zotz 1964
5. yoxkin 84
4. cumhu 1016
13. kankin 1520
14. mac 572
2. tzec 2576
13. xul 396
6. koyab 1076
9. zotz 452
8. yoxkin 300
4. mac 3420
10. pax 3700
15. pop 2152
11. mol 480
12. chen 2620
10. yoxkin 852
17. pax 1980
11. cumhu 2076
14. zotz 312
11. chen 1104
5. ceh 1468
15. zip 1888
8. mac 1112
14. mol 3240
13. zac 1244
8. ceh 2212
