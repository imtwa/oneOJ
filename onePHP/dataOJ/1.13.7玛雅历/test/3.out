186
4 akbal 169
12 lamat 887
10 mem 4998
1 canac 4942
4 ix 2123
8 kan 2387
6 mem 1612
3 caban 4980
11 muluk 1988
3 cimi 3414
12 ok 3521
12 ik 988
3 kan 3689
10 canac 3364
3 lamat 522
12 manik 1353
9 cib 982
2 caban 1583
13 cimi 5318
11 manik 881
11 canac 1550
12 caban 1298
8 ben 4392
2 caban 5071
9 chicchan 3240
9 cimi 399
2 akbal 4066
8 akbal 1466
7 ben 1354
1 eznab 2612
1 mem 2049
3 ix 1606
2 ik 5184
1 ben 1618
9 canac 5200
2 ok 348
3 manik 2987
6 ix 2909
13 muluk 1977
7 chicchan 708
2 eb 4812
7 caban 1011
10 ik 4061
4 eb 455
3 lamat 5245
8 mem 3392
10 kan 95
6 canac 1203
13 chuen 5318
10 chuen 4413
7 caban 1292
13 ix 106
10 eznab 724
5 ok 2449
10 akbal 4105
9 eb 3830
7 mem 3178
3 ben 4189
8 eb 3757
8 ix 994
6 manik 2863
4 muluk 4386
12 eb 3257
8 ik 4391
11 cib 157
8 eznab 1449
2 caban 1511
11 ok 483
11 lamat 4133
3 ix 2477
11 cib 62
3 akbal 2292
12 eb 1567
10 akbal 1606
2 chuen 1909
10 cib 2752
13 manik 3313
9 ix 3443
2 muluk 4133
1 manik 1551
9 ben 4391
7 cimi 1809
7 manik 4240
3 mem 4223
2 canac 1393
12 manik 3229
6 kan 3847
7 mem 2847
5 cib 5487
4 manik 1965
1 chuen 5498
11 cib 4695
4 ok 3437
8 lamat 3370
7 eznab 1966
7 ik 152
3 ben 1135
5 kan 4976
4 muluk 287
11 chicchan 1847
1 chicchan 321
2 cimi 887
7 eznab 5037
11 eznab 573
8 kan 4172
5 cib 820
6 chicchan 579
9 ben 2151
10 mem 3544
9 canac 1179
11 chicchan 2589
2 chuen 5262
2 cimi 4723
13 eznab 2903
12 chicchan 2364
7 ok 376
1 ik 3953
13 eb 3926
8 chicchan 2650
7 manik 4089
8 chuen 3745
9 lamat 2780
4 cimi 461
11 ik 3431
7 mem 2369
12 akbal 1106
3 ben 3639
1 ben 4594
1 eb 3835
3 akbal 5599
7 eznab 506
13 caban 2612
3 cimi 2212
11 ik 1864
2 chuen 3571
11 lamat 4813
2 manik 3291
5 canac 1348
12 eznab 3100
5 canac 505
13 ok 3425
4 kan 1478
9 canac 3937
2 akbal 5026
12 ik 2129
12 cib 3880
13 akbal 3381
12 cib 1915
4 eznab 5615
13 manik 1342
5 eb 3633
4 akbal 3887
3 muluk 3078
8 eb 2511
1 cimi 4611
12 chicchan 3066
11 ok 2409
1 eznab 1808
1 ix 309
10 cimi 1117
3 manik 2757
2 cimi 118
9 chicchan 1427
13 ix 2134
8 mem 803
7 akbal 3616
3 ix 556
12 manik 1511
2 ok 634
13 muluk 421
12 chicchan 4802
7 chuen 5195
10 cib 3021
8 eb 674
11 ben 3678
8 chuen 1196
10 eznab 2780
10 eb 2915
10 mem 438
2 eb 1550
4 cimi 2061
7 cib 2650
9 muluk 1562
2 mem 4549
2 ix 1747
10 muluk 3106
